### PR TITLE
Milestone 2 - Bookstore

### PR DESCRIPTION
**Pull request by Azeem838**
In this milestone, we accomplished the following:

1. Prepared the project directory with components, containers, actions and reducers.
2. Implemented the App Component
3. Implemented the BookList component that listed all the books in a table
4. Implemented the Book component that created a row for a book
5. Implemented the Book form component that displayed a form
6. Defined Redux actions and reducers
7. Initialized the Redux store and the initial state

Please find our commit below,
Thank you.

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
```
Comment (shloch) [inline src/components/BookList.js]: - this component is a "container" component and should be in the `src/containers/` folder

File: src/components/BookForm.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BookForm.js]: - this component is a "container" component and should be in the `src/containers/` folder

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Azeem838 , @PhillipUg 

Hello guys, everything is cool here.......this is awesome from you
The modification I will ask you to bring is with respect to requirements, nothing logical before we get final thumbs up

See comment below : 

Happy coding

**Comment by PhillipUg** [comment]
As recommended, we have fixed the following:
- moved `BookList` and `BookForm` components to the containers folder.

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All of the required changes were compliant with the required changes.
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**